### DISCUSSION TITLE
an option to remove feed

### DISCUSSION THREAD

**Discussion by madhums** [community/community#13255] [Other Feature Feedback, Questions, & Ideas]
Hi, it's been bothering me a little, I would like a way to **not** see any feed/recommendation when I land on github.com (home or dashboard whatever you call it). Instead of the feed I'd like issues and/or pull requests I've been contributing to. 

Would you be able to consider this?

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by madhums** [reply]
Ah! And this is not available for the "regular" feed?

**Reply by jfullstackdev** [reply]
yup, there is none

but we don't know what will happen, whether this will be combined as one dashboard with so many filters

**Comment by mikekavouras** [comment]
@madhums there's some ongoing research into making the homepage significantly more useful. I don't have many details but I can assure you it's top of mind for product at GH.

**Reply by jfullstackdev** [reply]
I think for me, this is really useful and the capability to disable it is good for those who claim that GitHub is not a social network, 

but my idea is that, there will be no more two tabs in that feed to separate seeing things, it's just the filter where we can see categories

and one useful addition is the personal dashboard like the Organization dashboard where you can see all of your commits and activities in the past days etc., and also, I was seeing requests in the past about those recommendations you don't want to see without blocking a user, but will block all related activities
